views/_includes/input-dates.pug

```diff
@@ -0,0 +1,28 @@
+mixin dateInputs(idPrefix, legendText, isoHintText, values)
```

Comment:
What happens if there is no `idPrefix`? Can we set a default value?

Reply:
ah, good catch

Reply:
oh wait, this actually won't be an issue because you have to pass something or a null to continue